tool/tctl/common/tctl.go

```diff
@@ -208,24 +208,10 @@ func TryRun(commands []CLICommand, args []string) error {
 		}
 	}
 
-	// Get license expired alerts.
-	alertCtx, _ := context.WithTimeout(ctx, constants.TimeoutGetClusterAlerts)
-	alerts, err := client.GetClusterAlerts(alertCtx, types.GetClusterAlertsRequest{
-		Labels: map[string]string{
-			types.AlertLicenseExpired: "yes",
-		},
-	})
-	if err != nil && !trace.IsNotImplemented(err) {
-		log.Warnf("getting cluster alerts: %v", trace.Wrap(err))
-	}
-
-	types.SortClusterAlerts(alerts)
-
-	for _, alert := range alerts {
-		if err := alert.CheckMessage(); err != nil {
-			log.Warnf("Skipping invalid alert %q: %v", alert.Metadata.Name, err)
-		}
-		fmt.Fprintf(os.Stderr, "%s\n\n", utils.FormatAlertOutput(alert))
+	if err := common.ShowClusterAlerts(ctx, client, os.Stdout, map[string]string{
```

Comment:
Isn't this one supposed to have a timeout?

Reply:
Also, is there a convenient way to check which command we've just executed? We could just have `tctl status` report on every alert, and skip the reporting here if that's the case - it would also simply the code by getting rid of the `ignore` map.

Reply:
The timeout is added in common.ShowClusterAlerts.